hardware/neopixel/neopixel.html

```diff
@@ -31,6 +31,16 @@
         <option value="grb">GRB</option>
     </select>
 </div>
+<div class="form-row">
```

Comment:
imho I'd prefer gamma after brightness if it's going to be default on and unlikely to be adjusted.

Reply:
I'll swap them around

Reply:
swapped